experimenter/experimenter/features/tests/test_load_feature_configs.py

```diff
@@ -75,12 +75,29 @@ def test_loads_new_feature_configs(self):
             sorted(["nimbus.test.string", "nimbus.test.int", "nimbus.test.boolean"]),
         )
 
+        self.assertEqual(
+            schema.set_pref_vars,
+            {
+                "string": "nimbus.test.string",
+                "int": "nimbus.test.int",
+                "boolean": "nimbus.test.boolean",
+            },
+        )
+
```

Comment:
Is there any way this can fail on us if we somehow pass in the wrong things to `set_pref_vars`? (I see we're checking if it's `None` in `load_feature_configs.py`)

Reply:
If its not JSON serializable it should throw and fail to import features.

The thing we're checking for none is here:

```python
sets_prefs = [
    _set_pref_name(v.set_pref)
    for v in feature.model.variables.values()
    if v.set_pref is not None  # <--
] 
```

Here if `v.set_pref` is None, it just means there is no `setPref:` entry for the variable, e.g.


```yaml
fooFeature:
  variables:
    bar:
      type: string
      setPref:
        branch: user
        pref: 'foo.bar.baz'
    qux:
      type: int
```

Here `bar` will have `set_pref = SetPref(branch="user", pref="foo.bar.baz")` and `qux` will have `set_pref = None`